src/openapi/streaming/streaming.ts

```diff
@@ -179,11 +181,17 @@ class Streaming extends MicroEmitter<EmittedEvents> {
         baseUrl: string,
         authProvider: AuthProvider,
         options?: Partial<types.StreamingConfigurableOptions>,
+        callbacks?: {
+            onProbe: (message: types.ProbeControlMessage) => void

```

Comment:
This could live just as tidily on the options object - wdyt?

We have that pattern already in `subscription.ts` too.

Reply:
Yeah I'd confused services options with this so thought it wouldn't be suitable but after checking I agree with you. Changes made